$NetBSD: patch-ab,v 1.3 2020/05/03 15:36:20 rillig Exp $

Remove one extra level of m4 quoting, so that the detection of stdio.h
functions works again.

https://sourceforge.net/p/awka/bugs/46/

Disable shared library. "It's unversioned and quite primitive, and
nothing is currently depending on it."

--- configure.orig	2001-07-11 06:00:52.000000000 +0000
+++ configure
@@ -2018,7 +2018,7 @@ cat > conftest.$ac_ext <<EOF
 #include <stdio.h>
 EOF
 if (eval "$ac_cpp conftest.$ac_ext") 2>&5 |
-  egrep "[[^v]]fprintf" >/dev/null 2>&1; then
+  egrep "[^v]fprintf" >/dev/null 2>&1; then
   :
 else
   rm -rf conftest*
@@ -2036,7 +2036,7 @@ cat > conftest.$ac_ext <<EOF
 #include <stdio.h>
 EOF
 if (eval "$ac_cpp conftest.$ac_ext") 2>&5 |
-  egrep "[[^v]]sprintf" >/dev/null 2>&1; then
+  egrep "[^v]sprintf" >/dev/null 2>&1; then
   :
 else
   rm -rf conftest*
@@ -2877,8 +2877,8 @@ rm -f conftest*
     EXE=''
     if test "$GCC" = yes
     then
-      SHARED_LIB='libawka.so'
-      SHAREDFLAG='-fPIC'
+      SHARED_LIB=''
+      SHAREDFLAG=''
     fi
   else
     EXE=$djgpp
